packages/components/src/localStorage/index.ts

```diff
@@ -0,0 +1,26 @@
+// import AsyncStorage from '@react-native-community/async-storage';
```

Comment:
put it under src/helper and rename the file to LocalStorageHelper

Reply:
done